app/models/event.rb

```diff
@@ -55,6 +56,10 @@ def deadline_has_passed?
     deadline < Date.current
   end
 
+  def startdate_has_passed?
+    startdate < Date.current
+  end
```

Comment:
This method is only used in your tests, nowhere else. Is that intentional? I don't really see how this adds the expected behaviour yet.

Reply:
This is from an earlier implementation that is actually not needed anymore.